### ISSUE TITLE
Error messages bug in grid display

### ISSUE THREAD

**Issue by kikky7** [pqina/filepond#766]
**Describe the bug**

When using grid display `.filepond--item { width: calc(25% - .5em); }` error messages are cut out (see example).

![image](https://user-images.githubusercontent.com/57031667/136655401-7f9243cb-83fa-4ff8-9a7e-0c1771f08c48.png)

There is no way to ignore invalid files (at least aren't any in docs) since I don't want to show errors in the list (due the bug with grid display), but in my own custom alert box... So I tried with this line of code `this.$refs.pond.removeFile(file);` (it's vuejs) under addFile event. It removes the element, but there is always short visible glitch of adding and removing that element and size animation of its container...

**Expected behavior**

Disable display of files with errors (don't add element at all to ul.filepond--list) or avoid glitch while removing the element.

**Information about your project:**

- Chrome latest
- W10
- FilePond version (4.28.2, vue-filepond 6.0.3)

**Comment by rikschennink** [comment]
This is correct, if the tile is too small the error message currently won't fit, best to scale up the tiles a bit till this has been resolved.
